test/server/cards/03_TWI/events/PyrrhicAssault.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Pyrrhic Assault', function () {
+    integration(function (contextRef) {
+        describe('Pyrrhic Assault\'s ability', function () {
+            it('should give each friendly unit, "When Defeated: Deal 2 damage to an enemy unit."', function () {
```

Comment:
nit: if you want you can remove the `describe` block and just put the whole name in the `it` since there's just one case